nuprojs/Microsoft.ServiceFabric.Actors/Microsoft.ServiceFabric.Actors.nuproj

```diff
@@ -148,12 +148,18 @@
       </File>
       <File Include="$(DropFolderNetStandard_Win)system.fabric.strings.dll">
         <TargetPath>build\netcoreapp2.0</TargetPath>
+      </File>
+	  <File Include="$(DropFolderNetStandard_Win)Microsoft.ServiceFabric.StartupServicesUtility.dll">
```

Comment:
Do you need to include StartupServicesUtility.dll in DropFolderNetStandard path as well ?

Reply:
In actor project when actor is build, it publish and put all required dlls at DropFolderNetStandard_Win 
From there this nuproj takes and put it under build\netcoreapp2.0

DropFolderNetStandard  is not required.